utils/util/src/main/scala/pl/touk/nussknacker/engine/util/loader/ModelClassLoader.scala

```diff
@@ -1,18 +1,28 @@
 package pl.touk.nussknacker.engine.util.loader
 
+import com.typesafe.scalalogging.LazyLogging
+
 import java.net.{URL, URLClassLoader}
 
 case class ModelClassLoader private(classLoader: ClassLoader,
-                                    urls: List[URL])
+                                    urls: List[URL]) {
+
+  override def toString: String = s"ModelClassLoader(${toString(classLoader)})"
 
-object ModelClassLoader {
+  private def toString(classLoader: ClassLoader): String = classLoader match {
```

Comment:
just curious, why do we need it instead of URL list?

Reply:
Because I want to see what exactly classloader was used (what hierarchy). Currently it shows `UrlClassloader@hash`. I have a  with compilation of scenario. After a while of digging I figure out that the problem was because compilation was done before topic schema registration - we are checking it not only on designer side, but also on runtime side. IMO it can be helpful in the future. In the past I have a few situations that I was leaving a breakpoint to see what is a hierarchy of those classloaders

Reply:
:+1: